tests/system_tests/appstart_systemtests.py

```diff
@@ -42,6 +41,12 @@ def setUpClass(cls):
 
         This depends on a properly set up docker environment.
         """
+        
+        SystemTests.base_image="appstart_systemtest_devappserver"
+        
+        utils.build_from_directory(os.path.dirname(devappserver_init.__file__),
+                           SystemTests.base_image,nocache=True)
```

Comment:
Please align the second argument with the first and put the nocache=True on a new line